chore: document tracing headers better (#12145)
PostHog/posthog.com: contents/docs/libraries/python/_snippets/contexts.mdx

Contexts can be associated with a session ID by calling `posthog.set_context_session`. Session IDs must be UUIDv7 strings.

```python
from posthog import new_context, set_context_session
with new_context():
    set_context_session(request.get_header("X-POSTHOG-SESSION-ID"))
```

<CalloutBox icon="IconInfo" title="Using PostHog on your frontend too?">

> If you're using the PostHog JavaScript Web SDK on your frontend, it generates a session ID for you and you can retrieve it by calling `posthog.get_session_id()`. You will then need to pass that session ID to your backend by setting the `X-POSTHOG-SESSION-ID` header on each fetch request. Alternatively you can enable '__add_tracing_headers' in your config and the header will be set automatically.
> You will need to extract the header in your request handler (if you're using our Django middleware integration, this happens automatically).

Reviewer: ```suggestion
If you're using the PostHog JavaScript Web SDK on your frontend, it generates a session ID for you and you can retrieve it by calling `posthog.get_session_id()`. You then need to pass that session ID to your backend by setting the `X-POSTHOG-SESSION-ID` header on each fetch request. Alternatively, you can enable '__add_tracing_headers' in your config and the header is set automatically.

You need to extract the header in your request handler (if you're using our Django middleware integration, this happens automatically).
```

Contexts can be associated with a session ID by calling `posthog.set_context_session`. Session IDs must be UUIDv7 strings.

> > **Using PostHog on your frontend too?** If you're using the PostHog JavaScript Web SDK on your frontend, it generates a session ID for you and you can retrieve it by calling `posthog.get_session_id()`. We then recommend passing that session ID to your backend by setting the `X-POSTHOG-SESSION-ID` header. Alternatively you can enable '__add_tracing_headers' in your config and the header will be set automatically.

Reviewer 2: This might look better as a callout box. I find that larger text chunks are better for callout boxes, one liners often do better as comment block + a emoji

https://posthog.com/handbook/content/docs-style-guide#callout-boxes
